code42.json

```diff
@@ -61,11 +61,129 @@
             "output": [],
             "versions": "EQ(*)"
         },
+        {
+            "action": "add departing employee",
+            "identifier": "add_departing_employee",
+            "description": "Add a departing employee to the departing employees detection list",
+            "verbose": "There is a short delay before you can query the updated results.",
+            "undo": "remove departing employee",
```

Comment:
I added these `undo` sections - they are neat. They make it so you can right click on old action and there is now an "undo" action available which just guides you through the opposite action

Reply:
that's neat!